libs/ledger-live-common/src/families/evm/react.react.test.ts

```diff
@@ -1,5 +1,8 @@
+/**
```

Comment:
I think that you can rename this file `react.test.js`
react.test.js pattern was used to apply @jest-environment jsdom